app/services/locale_service.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
```

Comment:
```suggestion

```

This magic comment isn't really needed in Rails apps anymore. All of the important upstream things made the necessary changes.

Reply:
really? I must have missed this. It's a ruby thing not a rails thing, is there a post or something explaining this?

Reply:
I forget where I heard it. Maybe at RubyConf in the Matz Q&A? 
It was something like, by Ruby encouraging this comment ahead of time, it encouraged gem makers and rails core to make the needed changes for the frozen string future. For individual apps it is mostly just extra noise.